cases/forms.py

```diff
@@ -51,7 +51,7 @@ class Meta:
     def clean_name(self):
         name = self.cleaned_data["name"]
         case = Case.objects.filter(name__iexact=name).exclude(id=self.instance.id)
-        if case:
+        if case.exists():
```

Comment:
checking `queryset.exists()` is more efficient than checking queryset truthiness [read more](https://django.doctor/advice/C9003)